packages/algob/test/internal/core/config/config-loading.ts

```diff
@@ -169,4 +190,79 @@ describe("config loading", function () {
       ).catch((err) => console.log(err)); ;
     });
   });
+
+  describe("load kmd accounts", function () {
+    useFixtureProject("config-project");
+    useEnvironment();
+
+    const fakeKmd: Kmd = {} as Kmd; // eslint-disable-line @typescript-eslint/consistent-type-assertions
+    const kmdOp = new KMDOperatorMock(fakeKmd);
+    let net: NetworkConfig;
+
+    beforeEach(function () {
+      net = this.env.config.networks.kmdNet;
+    });
+
+    afterEach(function () {
+      kmdOp.resetAccounts();
+    });
+
+    it("should ignore if kmd config is not defined", async function () {
+      net.kmdCfg = undefined;
+      const result = await _loadKMDAccounts(net, kmdOp);
+      assert.isUndefined(result);
+    });
+
+    it("should not connect to kmd if config is invalid", async function () {
+      const invalidKmdCfg = net.kmdCfg as KmdCfg;
+      invalidKmdCfg.port = 123; // invalid port
+
+      expectBuilderErrorAsync(
+        () => _loadKMDAccounts(net, new KMDOperator(createKmdClient(invalidKmdCfg))),
+        ERRORS.KMD.CONNECTION
```

Comment:
I don't think we should createKmdClient in tests. 
as this is not mock and we cannot test the correct tests for it (in test environment)
`this will always fail`